sktime/markov/msm/markov_state_model.py

```diff
@@ -133,16 +137,23 @@ def transition_matrix(self):
         """ The transition matrix on the active set. """
         return self._transition_matrix
 
+    @property
+    def transition_matrix_tolerance(self):
+        r""" The tolerance under which a matrix is considered a transition matrix. This means that all elements
+        must be non-negative and the row sums must be 1. """
+        return self._transition_matrix_tolerance
+
     def update_transition_matrix(self, value: np.ndarray):
         """ Sets the transition matrix and invalidates all cached and derived properties. """
         if value is None:
             raise ValueError("Markov state model requires a transition matrix, but it was None.")
         else:
             if not issparse(value):
                 value = np.asarray_chkfinite(value)
-            if not msmana.is_transition_matrix(value, tol=1e-8):
-                raise ValueError('The input transition matrix is not a stochastic matrix '
-                                 '(elements >= 0, rows sum up to 1).')
+            if not msmana.is_transition_matrix(value, tol=self.transition_matrix_tolerance):
```

Comment:
whats `msmana`

Reply:
an `msmtools` import, bound to change anyways

Reply:
cool